### ISSUE TITLE
Failing build due to deprecated config

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/deep-archive#171]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

https://github.com/NASA-PDS/deep-archive/actions/runs/10275100951/job/28433116329

### 🕵️ Expected behavior

I expected the build to be successful

### 📜 To Reproduce

```
DEBUG:pds.roundup.util:🏃‍♀️ Running «['python', 'setup.py', 'test']»
CRITICAL:pds.roundup.util:💥 Process with command line ['python', 'setup.py', 'test'] failed with status 1
CRITICAL:pds.roundup.util:🪵 Stdout = «»
CRITICAL:pds.roundup.util:📚 Stderr = «/github/workspace/venv/lib/python3.9/site-packages/setuptools/config/setupcfg.py:296: _DeprecatedConfig: Deprecated config in `setup.cfg`
!!

        ********************************************************************************
        The namespace_packages parameter is deprecated, consider using implicit namespaces instead (PEP 420).

        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
  parsed = self.parsers.get(option_name, lambda x: x)(value)
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'test'
```

**Comment by jordanpadams** [comment]
Closed per #172

**Comment by jordanpadams** [comment]
Skipping I&T since this is a build / technical issue
